template/frameworks/buefy/pages/index.vue

```diff
@@ -16,13 +17,16 @@
               <b-icon
                 :icon="feature.icon"
                 size="is-large"
-                type="is-primary"/>
+                type="is-primary"
+              />
             </div>
           </div>
-          <footer class="card-footer">
-            <div
+          <footer class="card-footer"><% if (eslint === 'yes') { %>
+            <!-- eslint-disable vue/no-v-html -->
+            <% } %><div
               class="card-footer-item"
-              v-html="feature.content"/>
+              v-html="feature.content"
```

Comment:
This cycle doesnt make much sense. In original it just inlined and its fine. Lets do same
https://github.com/buefy/buefy/blob/dev/docs/pages/Home.vue

Reply:
Fixed - separated repeatable parts to an individual `Card` component for sake of DRY.